Source/WebKit/NetworkProcess/cocoa/NetworkTaskCocoa.mm

```diff
@@ -239,6 +239,26 @@ static bool shouldCapCookieExpiryForThirdPartyIPAddress(const WebCore::IPAddress
     }
 }
 
+bool NetworkTaskCocoa::shouldBlockCookies(WebCore::ThirdPartyCookieBlockingDecision thirdPartyCookieBlockingDecision)
+{
+    return thirdPartyCookieBlockingDecision == WebCore::ThirdPartyCookieBlockingDecision::All;
+}
+
+std::pair<bool, WebCore::ThirdPartyCookieBlockingDecision> NetworkTaskCocoa::requestThirdPartyCookieBlockingDecision(const WebCore::ResourceRequest& request) const
```

Comment:
It might be clearer to separate `bool isOptInCookiePartitioningEnabled()` (which is only available under `HAVE(ALLOW_ONLY_PARTITIONED_COOKIES)`) and `WebCore::ThirdPartyCookieBlockingDecision requestThirdPartyCookieBlockingDecision`.